packages/driver/src/cy/commands/cookies.ts

```diff
@@ -110,7 +108,7 @@ export default function (Commands, Cypress, cy, state, config) {
     })
   }
 
-  const getAndClear = (log, timeout, options = {}) => {
+  const getAndClear = (log?, timeout?, options = {}) => {
```

Comment:
Why is log optional here? it's not optional for `automateCookies`

Reply:
Because it's called without them at line 164.

https://github.com/cypress-io/cypress/blob/bc87d6eae121cdbd31abd2c36ab1c788b36795b6/packages/driver/src/cy/commands/cookies.ts#L164